app/models/publication.rb

```diff
@@ -140,25 +140,6 @@ def build_from_sciencewire_hash(new_sw_pub_hash)
     self
   end
 
-  # this is a very temporary method to be used only for the initial import
-  # of data from CAP.
-  # @return [self]
-  def cutover_sync_hash_and_db
-    set_sul_pub_id_in_hash
-    pub_hash[:last_updated] = updated_at.to_s
-    add_all_db_contributions_to_my_pub_hash
-    # add identifiers that are in the hash to the pub identifiers db table
-    pub_hash[:identifier].each do |identifier|
-      publication_identifiers.create(
-        identifier_type: identifier[:type],
-        certainty: 'confirmed',
-        identifier_value: identifier[:id],
-        identifier_uri: identifier[:url])
-    end
-    update_formatted_citations
-    self
-  end
```

Comment:
On master
```
git grep 'cutover_sync_hash_and_db'
app/models/publication.rb:146:  def cutover_sync_hash_and_db
lib/tasks/cap_cutover.rake:507:    pub.cutover_sync_hash_and_db
lib/tasks/cap_cutover.rake:580:      pub.cutover_sync_hash_and_db
```
This PR proposes to remove the public method calls from `lib/tasks/cap_cutover.rake`.